integration/smartcontract/derp_test.go

```diff
@@ -0,0 +1 @@
+package smartcontract
```

Comment:
whats this

Reply:
ehe, that was the tx size unit test file that somehow didn't get deleted!